### PR TITLE
Doc rework

### PR DESCRIPTION
**Pull request by jerome-jutteau**

### CODE REVIEW

File: deploy/README.md [hunk 1]
```diff
@@ -1,58 +1,88 @@
...
+| Plugin Version | Minimal Kubernetes Version | Recommended Kubernetes Version |
+| -------------- | -------------------------- | ------------------------------ |
+| <= v0.0.10beta | 1.20                       | 1.23                           |
+
+# Configuration
+
+## Cluster Resource Tagging
+
+You must tag some cloud resources with a cluster identifier in order to allow Cloud Controller Manager to identify which resources are part of the cluster.
+This include:
```
Comment (outscale-mdr) [inline deploy/README.md:16]: Typo:  "This includes"

File: deploy/README.md [hunk 2]
```diff
@@ -1,58 +1,88 @@
...
+
+The tag key must be `OscK8sClusterID/my-cluster-id` (adapt `my-cluster-id`) and tag value can be one of the following values:
+- `shared`: resource is shared between multiple clusters, and should not be destroyed
+- `owned`: the resource is considered owned and managed by the cluster
+
+## Instances Tagging
+
+Additionally, instances must be tagged with their node name.
+
+Tag key is `OscK8sNodeName` and tag value `kubernetes.io/my-kybernetes-host-name` (adapt `my-cluster-id`).
```
Comment (outscale-mdr) [inline deploy/README.md:30]: The value is just `my-kybernetes-host-name` which should be the same as `kubernetes.io/hostname` computed

File: deploy/README.md [hunk 3]
```diff
@@ -1,58 +1,88 @@
...
+
+## Security Groups Tagging
+
+By default, the service controller will automatically create a Security Group for each Load Balancer Unit (LBU) and will attach it to nodes in a VPC setup.
+
+If you want to use a pre-created Security Group to be applied to be attached/associated to the LBU, you must tag it with key `OscK8sMainSG/my-cluster-id` and value `True`.
+
```
Comment (outscale-mdr) [inline deploy/README.md:37]: We could add a note for public cloud with the issue, what do you think ?

File: deploy/README.md [hunk 4]
```diff
@@ -1,58 +1,88 @@
...
+## Networking
+
+Node controller is deployed as a daemon set and will need to access [metadata server](https://docs.outscale.com/en/userguide/Accessing-the-Metadata-and-User-Data-of-an-Instance.html) in order to get informations about its node (cpu, memory, addresses, hostname).
```
Comment (outscale-mdr) [inline deploy/README.md:40]: Typo:  "information"

File: deploy/README.md [hunk 5]
```diff
@@ -1,58 +1,88 @@
...
+
+If you want more details about network configuration with OpenShift, check [openshift documentation](https://docs.openshift.com/container-platform/4.10/networking/understanding-networking.html).
+
+## Kubelet
+
+Kubelet must be run with `--cloud-provider=external`, (more details in [Cloud Controller Manager Administration](https://kubernetes.io/docs/tasks/administer-cluster/running-cloud-controller/#running-cloud-controller-manager) documentation).
+
+## Configuring cloud credentials
+
+Outscale Cloud Controller Manager need API access in order to create resources (like Load Balancer Units) or fetch some data.
```
Comment (outscale-mdr) [inline deploy/README.md:51]: Typo:  "needs"

File: deploy/README.md [hunk 6]
```diff
@@ -1,58 +1,88 @@
...
+
+It is recommanded to use a specific [Access Key](https://docs.outscale.com/en/userguide/About-Access-Keys.html) and create an [EIM user](https://docs.outscale.com/en/userguide/About-EIM-Users.html) with limited access. Check [EIM policy example](eim-policy.example.json) to apply to such EIM user.
```
Comment (outscale-mdr) [inline deploy/README.md:53]: Typo: "recommended"

File: deploy/README.md [hunk 7]
```diff
@@ -1,58 +1,88 @@
...
+More [helm options are available](../deploy/k8s-osc-ccm/README.md)
+
+# Check Deployment
+
+To check if Outscale Cloud Manager has been deployed, check for `osc-cloud-controller-manager`:
+```
+kubectl get pods -o wide -A -n kube-system | grep osc-cloud-controller-manager
 ```
 
+You can also deploy a simple application exposed by a Service like [2042 web application](../examples/2048/README.md).
```
Comment (outscale-mdr) [inline deploy/README.md:88]: Typo: "2048"

File: deploy/k8s-osc-ccm/README.md [hunk 8]
```diff
@@ -0,0 +1,14 @@
+This documentation provide details regarding Outscale Cloud Manager deployment through helm.
```
Comment (outscale-mdr) [inline deploy/k8s-osc-ccm/README.md:1]: Typo: "provides"

File: deploy/k8s-osc-ccm/README.md [hunk 9]
```diff
@@ -0,0 +1,14 @@
...
+
+| Parameter          | Description                              | Default                       |
+|--------------------|------------------------------------------|-------------------------------|
+| `replicaCount`     | k8s replicas                             | `1`                           |
+| `image.pullPolicy` | Container pull policy                    | `IfNotPresent`                |
+| `image.repository` | Container image to use                   | `outscale/cloud-provider-osc` |
+| `image.tag`        | Container image tag to deploy            | `v0.0.10beta`                 |
+| `imagePullSecrets` | Specify image pull secrets               | `[]`                          |
+| `podLabels`        | Labels for pod                           | `{}`                          |
+| `oscSecretName`    | Secret name containing cloud credentials |                               |
```
Comment (outscale-mdr) [inline deploy/k8s-osc-ccm/README.md:14]: Default value is "osc-secret"

File: docs/README.md [hunk 10]
```diff
@@ -0,0 +1,30 @@
+# Outscale Cloud Controller Manager (CCM)
+
+The Outscale Cloud Controller Manager (cloud-provider-osc) provides the interface between a Kubernetes cluster and 3DS Outscale service APIs. 
+This project allows a Kubernetes cluster to provision, monitor and remove Outscale resources necessary for operation of the cluster.
```
Comment (outscale-mdr) [inline docs/README.md:4]: I would have said "necessary for operating the cluster"

File: docs/README.md [hunk 11]
```diff
@@ -0,0 +1,30 @@
...
+
+More details on [cloud-controller role](https://kubernetes.io/docs/concepts/architecture/cloud-controller/) in Kubernetes architecture.
+
+# Features
+- Node controller: provides Kubernetes details about nodes (Outscale Virtual Machines)
+- Service controller: allow cluster user to expose Kubernetes Services using Outscale Load Balancer Unit (LBU)
```
Comment (outscale-mdr) [inline docs/README.md:10]: Typo: "allows"

File: examples/2048/README.md [hunk 12]
```diff
@@ -19,16 +19,12 @@ deployment.apps/2048-deployment   2/2     2            2           84s
...
-
-```
-
-- Cleanup resources:
-
-```
-/usr/local/bin/kubectl delete -f examples/2048/specs/
 ```
 
+In this example, you should now be able to access the service with your favorite browser on `http://ad4337ccf92644bc3ba83ee2d28dd9cf-588858383.eu-west-2.lbu.outscale.com:8383`.
```
Comment (outscale-mdr) [inline examples/2048/README.md:24]: the url can be retrieved with `kubectl get svc`

**Review by outscale-mdr — CHANGES_REQUESTED** [review]
The docs is better now :)

File: deploy/README.md (hunk 3 above)
Comment (jerome-jutteau) [reply deploy/README.md:38]: done :-)

**Review by outscale-mdr — APPROVED** [review]

**Pull request merged**